evo-delivery-app/evo-delivery-backend/src/database/models/driver.model.ts

```diff
@@ -0,0 +1,33 @@
+import mongoose,{Schema} from "mongoose";
+
+export interface IDriver{
+    _id?: mongoose.Types.ObjectId;
+    name: string;
+    maxCapacity: number;
+    maxLength: number;
+    createdAt: Date;
+    updatedAt: Date;
```

Comment:
In what way are these parameters reflected in the schema?
Are they being updated automatically?

Reply:
Yes. they create and update automatically by mongo